draft-ietf-jsonpath-iregexp.md

```diff
@@ -281,6 +286,13 @@ This document makes no requests of IANA.
 Security considerations
 =======================
 
+While technically out of scope of this specification, {{Section 10
+(Security Considerations) of -utf8}} applies to almost any practical
```

Comment:
I'd cut this back to just "applies to implementations".  I'm a bit allergic to unspecific language like "almost any" and "practical".